rest_framework/schemas/openapi.py

```diff
@@ -564,3 +583,27 @@ def _get_responses(self, path, method):
                 'description': ""
             }
         }
+
+    def _get_tags(self, path, method):
```

Comment:
Name this `get_tags()`. We'll document it, and expect folks to override.

Reply:
sure.

Reply:
I have done all the changes given by you and Tom.